pom.xml

```
@@ -346,7 +346,7 @@
         <groupId>com.github.tomakehurst</groupId>
         <artifactId>wiremock-jre8</artifactId>
         <scope>test</scope>
-        <version>${version.wiremock}</version>
+        <version>${version.com.github.tomakehurst}</version>
```

```suggestion
        <version>${version.com.github.tomakehurst-wiremock}</version>
```
No problem keeping the group here, but the project is known as `wiremock`. wdyt?

---

the convention is groupId[.artifactId] so if that is problem only possible new property name is `version.com.github.tomakehurst.wiremock-jre8` or `version.com.github.tomakehurst.wiremock`